sdk/core/Azure.Core/tests/HttpPipelineRequestContentTests.cs

```diff
@@ -98,6 +99,36 @@ public void ObjectContent()
             CollectionAssert.AreEqual(expected, destination.ToArray());
         }
 
+        [Test]
+        public void IntArrayContent()
+        {
+            var intArray = new int[] { 1, 2, 3 };
+            var expected = "[1,2,3]";
+            var destination = new MemoryStream();
+            var content = RequestContent.Create(intArray);
+
+            content.WriteTo(destination, default);
+            destination.Position = 0;
+            using var reader = new StreamReader(destination);
+
+            Assert.AreEqual(expected, reader.ReadToEnd());
+        }
+
+        [Test]
+        public void DictionaryContent()
+        {
+            var dictionary = new Dictionary<string, object> { { "keyInt", 1 }, { "keyString", "2" }, { "keyFloat", 3.1f } };
+            var expected = "{\"keyInt\":1,\"keyString\":\"2\",\"keyFloat\":3.1}";
```

Comment:
You can do something like [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/tests/public/JsonDataPublicMutableTests.cs#L108-L114) if you need to for net462.